microsoft/testsuites/kselftest/kselftest.py

```diff
@@ -0,0 +1,277 @@
+import os
+import re
+from dataclasses import dataclass
+from pathlib import PurePath, PurePosixPath
+from typing import Any, Dict, List
+
+from assertpy import assert_that
+
+from lisa import Environment, notifier
+from lisa.base_tools.uname import Uname
+from lisa.executable import Tool
+from lisa.messages import SubTestMessage, TestStatus, create_test_result_message
+from lisa.node import Node
+from lisa.operating_system import CBLMariner, Ubuntu
+from lisa.testsuite import TestResult
+from lisa.tools.git import Git
```

Comment:
Can you please merge tool imports together. You can do something like this: `from lisa.tools import Git, Lscpu, ...`